packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -1302,9 +2309,46 @@ class RoundSliderThumbShape extends SliderComponentShape {
       begin: sliderTheme.disabledThumbColor,
       end: sliderTheme.thumbColor,
     );
+    final double radius = radiusTween.evaluate(enableAnimation);
+
+    // Add a stroke of 1dp around the circle if this thumb would overlap
+    // the other thumb.
+    if (isOnTop) {
+      bool showValueIndicator;
+      switch (sliderTheme.showValueIndicator) {
```

Comment:
Is this value guaranteed to be non-null?

Reply:
Added asserts for the properties on the theme here